src/GitTfs/Core/DerivedGitTfsRemote.cs

```diff
@@ -225,12 +225,7 @@ public IFetchResult FetchWithMerge(int mergeChangesetId, bool stopOnFailMergeCom
             throw DerivedRemoteException;
         }
 
-        public void QuickFetch()
-        {
-            throw DerivedRemoteException;
-        }
-
-        public void QuickFetch(int changesetId)
+        public void QuickFetch(int changesetId, bool ignoreRestricted, bool printRestrictionHint)
```

Comment:
Is this refactoring OK? If yes, then I could make it an independent commit *before* the `--ignore-restricted-changeset` commit.